bridge/src/connection.rs

```diff
@@ -85,22 +85,29 @@ pub struct RpcParams {
     pub timeout_millis: Option<u64>
 }
 
-async fn create_client(host: String) -> Result<Client, ConnectionError> {
+#[derive(Debug, Clone)]
+pub struct ConnectionConfig {
+    pub identity: String,
+    pub client_name: String,
+    pub client_version: String
+}
```

Comment:
This doesn't seem to need to exist. You can just use core's `ClientOptions` directly.